main.rb

```diff
@@ -0,0 +1,51 @@
+require_relative './app'
+
+def main
+  app = App.new
+  loop do
+    display_menu
+    choose_action(app)
+  end
+end
+
+def display_menu
+  puts "\nWelcome to School Library App!"
+  puts "\nPlease choose an option by entering a number:"
+  options = [
+    '1 - List all books',
+    '2 - List all people',
+    '3 - Create a person',
+    '4 - Create a book',
+    '5 - Create a rental',
+    '6 - List all rentals for a given person id',
+    '7 - Exit'
+  ]
+  puts options
+end
+
+def choose_action(app)
+  choice = gets.chomp
+  case choice
+  when '1'
+    app.list_all_books
+  when '2'
+    app.list_all_people
+  when '3'
+    app.create_person
+  when '4'
+    app.create_book
```

Comment:
- [ ] Great work done on your project, kindly check, I get the error in the screenshot below when I try to create a book, seems you did not implement the `create_book` method.
![er-book](https://user-images.githubusercontent.com/48371156/212496338-6cc41c81-0205-486d-96d7-0b4f994aa2fb.png)